retrofit/src/main/java/retrofit/Callback.java

```diff
@@ -30,8 +30,8 @@
  */
 public interface Callback<T> {
   /** Successful HTTP response. */
-  void success(Response<T> response);
+  void onSuccess(Response<T> response);
```

Comment:
onResponse to match OkHttp? Success might imply 2xx only and this gets called for all responses.